src/Physics/physicsEngine.ts

```diff
@@ -33,6 +33,7 @@ export class PhysicsEngine implements IPhysicsEngine {
 
     /**
      * Creates a new Physics Engine
+     * Only AmmoJSPlugin allows soft bodies

```

Comment:
We should not have it here for a specific function but more on the documentation site.
We can also move it to the interface declaration of new functions like getBodyPressure

Reply:
Will do

Reply:
Done